.travis.yml

```diff
@@ -96,6 +96,10 @@ install:
     - python setup.py install
     - python -c "import mne; mne.sys_info()"
     - python -c "import numpy;print(numpy.__version__)"
+    - python -c "import scipy;print(scipy.__version__)"
+    - python -c "import pandas;print(pandas.__version__)"
```

Comment:
We just wanted to know the exact versions of one of the Travis builds - to be able to better investigate what's going on with the (inconsistently) failing tests.